src/chi/skeletons/alert.scss

```diff
@@ -0,0 +1,46 @@
+$base-color: set-color(grey, 15);
+$shine-color: set-color(grey, 10);
+$animation-duration: 2s;
+
+@mixin background-gradient {
+  background-image: linear-gradient(-45deg, $base-color 0, $shine-color 4rem, $base-color 8rem);
+  background-size: 75rem;
+}
+
+chi-alert:not(.hydrated) {
+  content: '';
+  display: flex;
+  font-size: 0;
+  width: 50rem;
+
+  &::before {
```

Comment:
Let's remove the before element and use a single 100% container. Can we reduce the delay between the skeleton disappearing and the component content appearing?

Reply:
I removed the before element.
Also adjusted height of cases when size="" is specified `chi-alert[size="sm"]` and `chi-alert[size='lg']`.
With the current approach we achieve animation and height of web components before CE is loaded (page doesn't jump), but I'm still trying to figure out how to reduce the gap between the animation and component content. I opened a question in Stencil slack Chat here: https://app.slack.com/client/T77MU2PJL/C78FYDJH3/thread/C78FYDJH3-1585141134.007100